ci/pipelines/pipeline.yml

```
@@ -0,0 +1,483 @@
+---
+resource_types:
+- defaults:
+    commit: ((.:commit))
+    repository_url: https://github.com/RoboJackets/apiary-mobile
+    token: (("github.com"/token.token))
+  name: github-check
+  source:
+    aws_access_key_id: ((aws/sts/ecr.access_key))
+    aws_region: us-east-1
+    aws_secret_access_key: ((aws/sts/ecr.secret_key))
+    aws_session_token: ((aws/sts/ecr.security_token))
+    repository: robojackets/concourse-github-check-resource
+  tags:
+  - resources
+  type: registry-image
+
+- name: pull-request
+  source:
+    repository: teliaoss/github-pr-resource
+  tags:
+  - resources
+  type: registry-image
+
+- name: github-webhook
+  defaults:
+    github_token: (("github.com"/token.token))
+    resources:
+      apiary-mobile:
+        events:
+        - push
+        github_uri: https://github.com/RoboJackets/apiary-mobile
+      pull-request:
+        events:
+        - push
+        - pull_request
+        github_uri: https://github.com/RoboJackets/apiary-mobile
+    webhook_token: ((webhook-token))
+  source:
+    aws_access_key_id: ((aws/sts/ecr.access_key))
+    aws_region: us-east-1
+    aws_secret_access_key: ((aws/sts/ecr.secret_key))
+    aws_session_token: ((aws/sts/ecr.security_token))
+    repository: robojackets/concourse-github-webhook-resource
+  tags:
+  - resources
+  type: registry-image
+
+- name: git-tag
+  type: registry-image
+  source:
+    repository: sarquella/concourse-git-tag-resource
+  tags:
+    - resources
+
+- defaults:
+    url: ((slack-webhook))
+  name: slack
+  source:
+    repository: mockersf/concourse-slack-notifier
+  tags:
+  - resources
+  type: registry-image
+
+resources:
+- check_every: 1h0m0s
+  icon: android
+  name: android-build-box
+  public: true
+  source:
+    repository: mingc/android-build-box
+    tag: 1.24.0
+  tags:
+  - resources
+  type: registry-image
+
+- icon: github
+  name: apiary-mobile
+  public: true
+  source:
+    branch: main
+    password: (("github.com"/token.token))
+    uri: https://github.com/RoboJackets/apiary-mobile
+    username: x-access-token
+  tags:
+  - resources
+  type: git
+  webhook_token: ((webhook-token))
+
+- icon: source-pull
+  name: pull-request
+  public: true
+  source:
+    access_token: (("github.com"/token.token))
+    base_branch: main
+    repository: RoboJackets/apiary-mobile
+    disable_forks: true
+  tags:
+  - resources
+  type: pull-request
+  webhook_token: ((webhook-token))
+
+- name: tagged-release
+  type: git-tag
+  icon: tag
+  source:
+    uri: https://github.com/RoboJackets/apiary-mobile
+    username: x-access-token
+    password: (("github.com"/token.token))
+    tag_filter: v*
+  tags:
+    - resources
+
+- name: tests-check
+  type: github-check
+  icon: check
+  public: true
+  source:
+    check_name: Apiary Android Tests
```

I would personally just label this 
```suggestion
    check_name: Tests
```